lms/templates/courseware/courseware.html

```diff
@@ -102,6 +102,10 @@
 
 </%block>
 
+<%block name="header_bar">
+    <p>Header!!!</p>
```

Comment:
I'm assuming this will get overridden elsewhere, but is this the placeholder text we want?